.devcontainer/cuda12.9-conda/devcontainer.json

```diff
@@ -5,21 +5,21 @@
     "args": {
       "CUDA": "12.9",
       "PYTHON_PACKAGE_MANAGER": "conda",
-      "BASE": "rapidsai/devcontainers:25.12-cpp-mambaforge"
```

Comment:
Yikes, looks like the merge commit added a bunch of ci, packaging and cmake changes that need to be reverted.